packages/bpk-component-boilerplate/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "bpk-component-boilerplate",
+  "version": "0.0.0",
+  "description": "Backpack boilerplate component. For reference only, not intended to be used or published.",
```

Comment:
> For reference only, not intended to be used or published.

I bet that this copy gets accidentally published in a new component 😏

Reply:
This can be removed once my super new CLI is up and running.